apps/src/templates/courseOverview/CourseEditor.js

```diff
@@ -58,18 +58,71 @@ export default class CourseEditor extends Component {
       courseFamilies,
       versionYearOptions
     } = this.props;
+    const urlPath = `/courses/${name}`;
     return (
       <div>
         <h1>{name}</h1>
         <label>
-          Title
+          Display Name
           <input
             type="text"
             name="title"
             defaultValue={title}
             style={styles.input}
           />
         </label>
+        <label>
+          Course URL
+          <input
+            type="text"
+            value={urlPath}
+            style={styles.input}
+            disabled={true}
+          />
+        </label>
+        <label>
```

Comment:
in my text editor I moved the "publish settings" down... but github shows it as me moving whatever was below it up to here. 🤷